modules/gapi/include/opencv2/gapi/gcomputation.hpp

```diff
@@ -172,7 +172,7 @@ class GAPI_EXPORTS_W GComputation
      * @param in input GMat of the defined unary computation
      * @param out output GMat of the defined unary computation
      */
-    GComputation(GMat in, GMat out);                   // Unary overload
+    GAPI_WRAP GComputation(GMat in, GMat out);                   // Unary overload
```

Comment:
Please keep two whitespaces for comments: on this line
```cpp
GAPI_WRAP GComputation(GMat in, GMat out);  // Unary overload
```

> Also, lines that are non-obvious should get a comment at the end of the line. These end-of-line comments should be separated from the code by 2 spaces. Example:

source: https://google.github.io/styleguide/cppguide.html#Implementation_Comments

Reply:
Fixed